controllers/filmsController.js

```diff
@@ -0,0 +1,11 @@
+export default `SELECT film_id,
+  film_title,
+  release_date,
+  director_name,
+  imdb_rating,
+  genre
+FROM films f
+  INNER JOIN directors d
```

Comment:
Just wondering why you did an Inner Join. Not saying there is an issue.

Reply:
I did an inner join just to get the directors' names instead of the ID. Did you think we should change it to another type of JOIN?

Reply:
No that makes sense !!